packages/xml-views-validation/test/validators/element/use-of-deprecated-class-spec.ts

```diff
@@ -0,0 +1,144 @@
+import { expect } from "chai";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import {
+  GEN_MODEL_TIMEOUT,
+  generateModel
+} from "@ui5-language-assistant/test-utils";
+import { testValidationsScenario } from "../../test-utils";
+import { validateUseOfDeprecatedClass } from "../../../src/validators/elements/use-of-deprecated-class";
+
+describe("the use of deprecated class validation", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+
+  before(async function() {
+    this.timeout(GEN_MODEL_TIMEOUT);
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+  });
+
+  context("true positive scenarios", () => {
+    it("will detect usage of a deprecated class", () => {
```

Comment:
Add a test where there is no closing tag (so we know that the warning is returned for invalid syntax too). And maybe one where the tag itself isn't closed (`<Button`)?

Reply:
I add a single test where the tag is not closed `<Button`
because it is a **functional scenario** where a user is typing manually (without content assist)
and we want to give the user feedback ASAP so its worth testing it.

However I don't want to test too many invalid syntax scenarios in validations flow, this is the concern of the AST package in XML-Tools